src/scenes/Modals/ConfirmLeaveModal.tsx

```diff
@@ -56,6 +57,7 @@ const LeaveWrapper = styled.div`
   flex-direction: column;
   justify-content: center;
   place-items: center;
+  height: 100%;
```

Comment:
this page is already messed up on mobile; this just makes sure the full-page version isn't a regression

| Before | After |
| ----- | ----- |
| ![Screen Shot 2022-05-29 at 11 09 15 PM](https://user-images.githubusercontent.com/12162433/170910850-442963aa-5814-4f8d-bdab-da074084fcf7.png) | ![Screen Shot 2022-05-29 at 11 09 19 PM](https://user-images.githubusercontent.com/12162433/170910849-a3f512fa-ef8d-4e10-b511-72b9c70fe796.png) |